openshift/ci-operator/source-image/Dockerfile

```diff
@@ -3,3 +3,6 @@ FROM src
 COPY oc /usr/bin/oc
 COPY --from=registry.svc.ci.openshift.org/openshift/knative-v0.17.3:knative-serving-src /go/src/knative.dev/serving/ /go/src/knative.dev/serving/
 COPY --from=registry.svc.ci.openshift.org/openshift/knative-v0.17.2:knative-eventing-src /go/src/knative.dev/eventing/ /go/src/knative.dev/eventing/
+COPY --from=registry.svc.ci.openshift.org/openshift/knative-v0.17.1:knative-eventing-contrib-src /go/src/knative.dev/eventing-contrib/ /go/src/knative.dev/eventing-contrib/
+
+RUN chmod g+w /go/src/knative.dev/serving/ && chmod g+w /go/src/knative.dev/eventing/ && chmod g+w /go/src/knative.dev/eventing-contrib/
```

Comment:
Why is this needed and wasn't before?

Reply:
In Knative Kafka, we have a problem that we aren't able to write anything to `/go/src/knative.dev/eventing-contrib`.
This is required for tests.

cc @mgencur